pep-0590.rst

```diff
@@ -155,12 +155,12 @@ Third-party built-in classes using the new extended call interface
 ------------------------------------------------------------------
 
 To enable call performance on a par with Python functions and built-in functions, third-party callables should include a ``vectorcall`` function pointer
-and set ``tp_vectorcall`` to the correct value.
-Any class the sets ``tp_vectorcall`` to non-zero should also implement the ``tp_call`` function and make sure its behaviour is consistent with the ``vectorcall`` function.
+and set ``tp_vectorcall_offset`` to the correct value.
+Any class that sets ``tp_vectorcall`` to non-NULL should also implement the ``tp_call`` function and make sure its behaviour is consistent with the ``vectorcall`` function.
```

Comment:
"`tp_vectorcall` to non-NULL" should read "`tp_vectorcall_offset` to non-zero"

Reply:
Done